src/clj/rems/email/core.clj

```diff
@@ -74,9 +74,8 @@
         email (assoc email-spec
                      :from (:mail-from env)
                      :to (or (:to email-spec)
-                             (:email
-                              (users/get-user
-                               (:to-user email-spec)))))
+                             (:email (user-settings/get-user-settings (:to-user email-spec)))
+                             (:email (users/get-user (:to-user email-spec)))))
```

Comment:
hmm, so we use the email from user settings here, but not when rendering user details? I guess that's an ok design choice but it could be made more explicit

Reply:
Where are we rendering user details?

Reply:
e.g. applicant details in applications, selecting users as reviewers, blacklist tables